config/routes.rb

```diff
@@ -64,6 +64,7 @@
 
   resources :organizations
   resources :projects
+  get '/projects/archive/:id', to: 'projects#archive'
```

Comment:
Such types of actions should be done through POST method. GET is a tricky thing as users could accidentally trigger it.